sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Bias-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1980 @@
+{
```

Comment:
<p>Can you clear all outputs? It makes the notebooks cleaner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks for the feedback. The notebooks take more than one hour to run, we keep the outputs on purpose so that the customers don't have to spend too much time if they just want to see what the product does.

Reply:
Per offline discussion, the outputs have been cleared for this PR. Later I will initiate a discussion for the guidance on notebook outputs. Thanks.